templates/oauth2/oauth2-login.html.twig

```diff
@@ -0,0 +1,144 @@
+
+{#
+ # Authorization Server Member
+ #
+ # @package   OpenEMR
+ # @link      http://www.open-emr.org
+ # @author    Jerry Padgett <sjpadgett@gmail.com>
+ # @copyright Copyright (c) 2020 Jerry Padgett <sjpadgett@gmail.com>
+ # @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+#}
```

Comment:
don't forget to add yourself.

Reply:
You don't want full responsibility anymore eh?

Reply:
not at all. credit where credit is due....

Reply:
Fixed.